Gemfile.lock

```diff
@@ -19,28 +19,28 @@ GEM
     execjs (2.7.0)
     faraday (0.11.0)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.17)
+    ffi (1.9.18)
```

Comment:
Yes exactly. When I did `bundle install` after checking out the project. It is mostly influenced by the fact the `github-pages` got upgraded and altered its own dependencies. The `Gemfile.lock` file should not be much inspected as it is a derivative of the `Gemfile`—it is the equivalent of shrinkwrap file for npm.